test/FsAutoComplete.IntegrationTests/Tooltips/output.json

```diff
@@ -207,7 +207,7 @@
     [
       {
         "Signature": "val map: \n   mapping: 'T -> 'U ->\n   list   : 'T list  \n         -> 'U list",
-        "Comment": "Builds a new collection whose elements are the results of applying the given function\n to each of the elements of the collection.\n\n`list`: The input list.\n`mapping`: The function to transform elements from the input list.\nGeneric parameters:\n`'T` is `int`\n`'U` is `int`",
+        "Comment": "",
```

Comment:
Hmm, this shouldn't be changed, right?

Reply:
I'll undo it, but the test fails before my changes because this is empty.